Assets/GothicVR/Scripts/Npc/Routine.cs

```diff
@@ -7,44 +7,65 @@
 using GVR.Phoenix.Util;
 using GVR.Util;
 using GVR.World;
+using System;
+using System.Data;
 
 namespace GVR.Npc
 {
     public class Routine : MonoBehaviour
     {
-        private static readonly int SPEED = 10;
-        private GameTime gameTime;
+        private const float SPEED = 1f;
+        private GameTime gameTime = new();
+        PxCs.Data.WayNet.PxWayPointData waypoint;
+        RoutineData routine;
```

Comment:
This is the cached "currently active routine", righ? I recommend you name it like this. "currentRoutine" or "activeRoutine". This eases reading in the future.